src/components/AppProvider/AppProvider.tsx

```diff
@@ -36,7 +25,7 @@ export default class AppProvider extends React.Component<Props> {
   private scrollLockManager: ScrollLockManager;
   private subscriptions: {(): void}[] = [];
 
-  constructor(props: Props) {
+  constructor(props: AppProviderProps) {
```

Comment:
Should we keep this `Props` to stay consistent with the generic on `L21`
```suggestion
  constructor(props: Props) {
```